static/app/views/performance/landing/widgets/components/accordion.tsx

```diff
@@ -41,14 +48,31 @@ function AccordionItem({
   children,
   setExpandedIndex,
   content,
+  replayAccordion,
 }: {
   children: ReactNode;
   content: ReactNode;
   currentIndex: number;
   isExpanded: boolean;
   setExpandedIndex: (index: number) => void;
+  replayAccordion?: boolean;
 }) {
-  return (
+  return replayAccordion ? (
```

Comment:
Maybe this condition is better named something like `buttonOnLeft={true}`? That way it's not a specific replay thing, and other cases can decide if they want buttons on left or right.